static/app/views/starfish/components/chartPanel.tsx

```diff
@@ -20,19 +22,27 @@ export default function ChartPanel({title, children, button}: Props) {
             {button}
           </Header>
         )}
+        {subtitle && (
+          <SubtitleContainer>
+            <Subtitle>{subtitle}</Subtitle>
```

Comment:
This change also means the other usages of the chart panel across starfish can add a subtitle as well. I can take care of handling that.